source/feature/feature-template.html.md

```diff
@@ -37,7 +37,7 @@ Include you email address that you can be reached should people want to contact
 Expand on the summary, if appropriate. Explain the goal; the more details you can provide the better. 
 
 ### Prerequisites
-List any hardware or software prerequisites. Or any steps that need to be performed before the feature can be implemented.
+List any hardware or software prerequisites or any steps that need to be performed before the feature can be implemented.
```

Comment:
"**and** any steps is more curate as both  hardware \ software prerequisites and future steps must be listed.